basics_of_sst-core/tutorial/Example00.md

```diff
@@ -0,0 +1,568 @@
+# The Simplest Simulation
+
+All SST simulations consist of components and subcomponents that pass messages over connecting links.  So at its most basic, the simplest possible SST simulation consists of a single component that performs some action.
+
+This is the first example to be implemented, a single component that will count to a specified limit and end.
+
+Navigate to your top level SST folder and create a folder named `example`.  Then navigate to the folder you just created and create another folder named `Example00`.  This folder will contain all the files necessary to implement the simulation.
+
+## Defining a Component
+
+Navigate to the Example00 folder.  In the editor of your choice, create a new file and enter the code show below.  Save it as `ExampleComponent.h`.
+
+```
+#ifndef MY_COMPONENT_H
+#define MY_COMPONENT_H
+
+#include <sst/core/component.h>
+#include <sst/core/output.h>
+#include <sst/core/params.h>
+
+namespace Example00
+{
+    // Define the log levels.  Each level displays its own message and the
+    // levels above it.
+    //
+    // For example, ALL displays all log messages, INFO displays INFO, WARN,
+    // and FATAL messages, FATAL displays only FATAL messages.
+    //
+    const uint64_t FATAL = 0;
+    const uint64_t WARN  = 1;
+    const uint64_t INFO  = 2;
+    const uint64_t DEBUG = 3;
+    const uint64_t TRACE = 4;
+    const uint64_t ALL   = 5;
+
+    // This is a very simple component.  It only registers a clock and 
+    // prints log messages as the clock handler is called.
+    //
+    // Remember, all components inherit from SST::Component
+    //
+    class ExampleComponent : public SST::Component
+    {
+        public:
+            // Constructor/Destructor
+            //
+            ExampleComponent(SST::ComponentId_t id, SST::Params &params);
+            ~ExampleComponent() {}
+
+            // Standard SST::Component functions.  These all need to
+            // be implemented in the component, even if they are empty.
```

Comment:
Mentioned in ExampleComponent.cc: These functions do not need to be implemented if empty.